.github/workflows/fly-deploy.yml

```diff
@@ -0,0 +1,18 @@
+# See https://fly.io/docs/app-guides/continuous-deployment-with-github-actions/
+
+name: Fly Deploy
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy:
+    name: Deploy app
+    runs-on: ubuntu-latest
+    concurrency: deploy-group    # optional: ensure only one action runs at a time
+    steps:
+      - uses: actions/checkout@v4
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: flyctl deploy --remote-only
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
```

Comment:
just a question is it dummy reference ir you are using any secrets storage for the token ?

Reply:
I use my private 1passord for that and since I am the administrator of the repo, I can setup that into github. 
Quick note, I will remove this file, it is not part of this task to automatic deploy the application. I prefer to keep manual deploys because the account is using my private Credit Card.